rules/solid/tests/Rector/If_/ChangeAndIfToEarlyReturnRector/Fixture/if_in_foreach.php.inc

```diff
@@ -42,6 +42,7 @@ class IfInForeachClass
                 return;
             }
             $this->processNameOrIdentifier($node);
+            return;
```

Comment:
we are loosing a empty line before the `return;`. not sure this is a problem.

Reply:
Spaces are handled by coding standard tools, not a problem here